### PR TITLE
mp-toggle fixes for non-chrome browsers

### PR DESCRIPTION
**Pull request by tdumitrescu**
@jbwyme opened this against yr styleguide branch for the real diff

### CODE REVIEW

File: lib/components/toggles/index.js [hunk 1]
```diff
@@ -14,10 +14,12 @@ registerMPComponent('mp-toggle', class extends Component {
 
       helpers: {
         selectOption: value => this.value = value,
-        toggleOptions: () => Array.from(this.querySelectorAll('mp-toggle-option')).map(el => ({
```
Comment (jbwyme) [inline lib/components/toggles/index.js]: this doesn't work?

File: lib/components/toggles/index.styl [hunk 2]
```diff
@@ -1,6 +1,7 @@
+@import '../../../stylesheets/common'
 @import '../../../stylesheets/defaults'
 
-:host
++host-styles(mp-toggle)
```
Comment (tdumitrescu) [inline lib/components/toggles/index.styl]: without the +? tried, didn't work. stylus is so terrible with silent failures

File: lib/components/toggles/index.js (hunk 1 above)
Comment (tdumitrescu) [inline lib/components/toggles/index.js]: only in native implementation... https://github.com/mixpanel/mixpanel-common/pull/65/commits/3052b74036289b17cdeb18b7872013284ff31be5

File: lib/components/toggles/index.styl (hunk 2 above)
Comment (tdumitrescu) [inline lib/components/toggles/index.styl]: oh you mean without the display stuff. no i need `inline-block` here not `inline-flex`

File: lib/components/toggles/index.js (hunk 1 above)
Comment (jbwyme) [inline lib/components/toggles/index.js]: 😷

**Comment by tdumitrescu** [comment]
merged to master

**Pull request closed without merging**